keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ResourceRepositoryTests.kt

```diff
@@ -225,111 +219,6 @@ abstract class ResourceRepositoryTests<T : ResourceRepository> : JUnit5Minutests
     }
   }
 
-  data class CheckLockFixture<T : ResourceRepository>(
```

Comment:
Moved elsewhere (into the new tests that are common with `DeliveryConfigRepository`)